crypto/ec/ecp_nistp384.c

```diff
@@ -0,0 +1,1975 @@
+/*
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* Copyright 2023 IBM Corp.
```

Comment:
/*
 * Copyright

Reply:
I don't see why not.  There are plenty of other files with dual copyrights.